src/View/Helper/PaginatorHelper.php

```diff
@@ -1106,4 +1159,21 @@ public function implementedEvents()
     {
         return [];
     }
+
+    /**
+     * Helper method to deprecate the 'model' option throughout PaginationHelper.
+     *
+     * @deprecated 3.4.0
+     * @param array $options Array of options
+     * @return array Array of options
```

Comment:
Either here or in the long description of this method you could/should document the intent a little bit more.
A one liner should suffice. What is been done to the model option to deprecate it?